.github/workflows/pull-request-robot.yml

```diff
@@ -31,5 +31,5 @@ jobs:
         uses: actions/labeler@v4
         with:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
-          configuration-path: .github/actions/labeler.yml
+          configuration-path: ../actions/labeler.yml
```

Comment:
You need a `actions/checkout` to check out the source codes first

Reply:
I'll fix it.